web/styles/right_sidebar.css

```diff
@@ -214,6 +214,7 @@ $user_status_emoji_width: 24px;
 
 .my_user_status {
     opacity: 0.5;
+    white-space: nowrap;
```

Comment:
`white-space: nowrap;` prop is added so that "(you)" in pill don't break. not for right sidebar


![image](https://github.com/zulip/zulip/assets/73663475/c3ccda36-8825-41f4-a373-defc886e9474)

Reply:
Yeah, I just wanted to make sure that this does not break the tooltip in right sidebar because same class is used there as well.